### ISSUE TITLE
New check which find duplicate except clauses

### ISSUE THREAD

**Issue by kasium** [PyCQA/flake8-bugbear#254]
flake8-bugbear already find duplicate exeptions in the same tuple, but across different except's this is not done

**Examples**:
```python
try:
    pass
except ValueError:
    pass
except ValueError:
    pass
```
```python
try:
    pass
except (ValueError, TypeError):
    pass
except (ValueError, UnicodeError):
    pass
```

I'm happy tp contribute this check 😄

**Comment by cooperlees** [comment]
I guess this is doable, so sure, would accept this. Should be easy to make no noisey.

Please add description to README and add unit tests for the new checks please for it to be accepted.

**Comment by kasium** [comment]
Thanks for your comment. Let me give it a try

**Comment by cooperlees** [comment]
Released in 22.9.11
